src/Plugins/Init.php

```diff
@@ -98,4 +106,68 @@ private function init(): void
 
         exit(0);
     }
+
+    private function initLaravel(): int
+    {
+        View::render('components.badge', [
+            'type' => 'INFO',
+            'content' => 'Laravel installation detected, pest-plugin-laravel will be installed.',
+        ]);
+
+        exec('composer require pestphp/pest-plugin-laravel 2.x-dev', result_code: $result);
+
+        /** @var int $result */
+        if ($result > 0) {
+            View::render('components.badge', [
```

Comment:
Remove this.

Reply:
the updated code already prints error output, so no need to do here too